doc/ADR/homebrew_tap.md

```diff
@@ -0,0 +1,71 @@
+# Homebrew tap instructions
+
+## User instructions
+
+1. Add the CBMC tap using the following command:
+
+   ```sh
+   $ brew tap diffblue/cbmc
+   ```
+
+2. Install any of the available CBMC versions in the following manner:
+
+   ```sh
+   $ brew install diffblue/cbmc/cbmc@5.54.0
+   ```
+
+## Developer instructions
+
+1. First of all, you need to create a new tap locally:
```

Comment:
Under what circumstances do we need to create a tap locally? Is this part of creating an entirely separate tap on github, or modifying the tap that already exists to add new versions?

Reply:
Yeah that was there just for documentation purposes.

Most likely you won't need to do that

It's a one off for setting up a new tap, and as we already have one, (it's highly likely) no one's going to need to do that again.